schema/2.0.0/corpora_schema.md

```diff
@@ -0,0 +1,852 @@
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td>categorical with <code>str</code> categories. This MUST be an EFO term.<br><br>An assay based on 10X Genomics products SHOULD either be <a href="http://www.ebi.ac.uk/efo/EFO_0008995"><code>"EFO:0008995"</code></a> for <i>10x technology</i> or <b>preferably</b> its most accurate child. Other assays SHOULD be the most accurate child of either <a href="http://www.ebi.ac.uk/efo/EFO_0002772">EFO:0002772</a> for <i>assay by molecule</i> or <a href="http://www.ebi.ac.uk/efo/EFO_0010183">EFO:0010183</a>  for <i>single cell library construction</i>.<br><br>
+        If there is not an exact match for the assay, clarifying text MAY be enclosed in parentheses and appended to the most accurate term. For example, the sci-plex assay could be curated as <code>"EFO:0010183 (sci-plex)"</code>.<br><br>Recommended values for specific assays:
+          <br><br>
+          <table>
+          <thead>
+          <tr>
+          <th>For</th>
+          <th>Use</th>
+          </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><i>10x 3' v2</i></td>
+              <td><a href="http://www.ebi.ac.uk/efo/EFO_0009899"><code>"EFO:0009899"</code></a></td>
+            </tr>
+            <tr>
+              <td><i>10x 3' v3</i></td>
+              <td><a href="http://www.ebi.ac.uk/efo/EFO_0009922"><code>"EFO:0009922"</code></a></td>
+            </tr>
+            <tr>
+              <td><i>10x 5' v1</i></td>
+              <td><a href="http://www.ebi.ac.uk/efo/EFO_0011025"><code>"EFO:0011025"</code></a></td>
+            </tr>
+            <tr>
+              <td><i>Smart-seq</i></td>
+              <td><a href="http://www.ebi.ac.uk/efo/EFO_0008930"><code>"EFO:0008930"</code></a></td>
+            </tr>
+            <tr>
+              <td><i>Smart-seq2</i></td>
+              <td><a href="http://www.ebi.ac.uk/efo/EFO_0008931"><code>"EFO:0008931"</code></a></td>
+            </tr>
+          </tbody></table>
+        </td>
+    </tr>
+</tbody></table>
+<br>
+
+### cell_type_ontology_term_id
+
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>cell_type_ontology_term_id</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Curator</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td>categorical with <code>str</code> categories. This MUST be a CL term. 
+        </td>
+    </tr>
+</tbody></table>
+<br>
+
+### development_stage_ontology_term_id
+
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>development_stage_ontology_term_id</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Curator</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td>categorical with <code>str</code> categories. If unavailable, this MUST be <code>"unknown"</code>.<br><br>If <code>organism_ontolology_term_id</code> is <code>"NCBITaxon:9606"</code> for <i>Homo sapiens</i>, this MUST be the most<br>accurate HsapDv term with the following STRONGLY RECOMMENDED:
```

Comment:
If you read this statement leading into the **Use** column in the table below, it reads that it is STRONGLY RECOMMENDED that it MUST be...
Same with the mouse table for the property

Reply:
It is ordered and logical as I read it:

* MUST be the most accurate HsapDv term
* with the following STRONGLY RECOMMENDED

Reply:
...but then you STRONGLY RECOMMEND (it) "MUST be a term from the set of Carnegie stages 1-23"

Reply:
Doh. Thanks for catching this.

OK - I'll rewrite to STRONGLY RECOMMEND "~~MUST be a~~ A term ..."

Reply:
Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/42bd59f857767044302b378e84c9b05be29fbd4b